packages/teleport/src/lib/tdp/codec.test.ts

```diff
@@ -104,7 +104,9 @@ test('decodes message types', () => {
   const { buffer: pngFrameBuf } = makeBuf(MessageType.PNG_FRAME);
   const { buffer: clipboardBuf } = makeBuf(MessageType.CLIPBOARD_DATA);
   const { buffer: errorBuf } = makeBuf(MessageType.ERROR);
-  const { buffer: invalidBuf } = makeBuf(MessageType.MFA_JSON + 1);
+  const { buffer: invalidBuf } = makeBuf(
```

Comment:
I noticed this is different in another PR so just a note to make sure they land in order to get the correct implementation for the `invalidBuf`